python-frontend/typeshed_serializer/checksum

```diff
@@ -1,2 +1,2 @@
-b0f9d8a6ebbbbe82ab815781dbff772d320ba6f67915c2cef80308dc6d209251
-6c15956136fdf02bde39d93ead39411a3c7908e21b1a8250ed3600e2bde1430e
\ No newline at end of file
+51ede4a5b5795ebd7cf3a0201643b16bef8ba617136121af51bcab09746d12a5
+40539c4a5c33308396640492f6bb790585ad37fb444d77404018dc838bef47e9
```

Comment:
Checksums don't match on my environment, I will retry it anyway with a fresh one